.circleci/config.yml

```diff
@@ -15,25 +15,24 @@ executors:
     - image: circleci/php:<< parameters.php-version >>-browsers
       environment:
         CIRCLE_EXECUTOR: stack
-    - image: docker.elastic.co/elasticsearch/elasticsearch:7.6.0
+    - image: docker.elastic.co/elasticsearch/elasticsearch:7.7.0
       name: elasticsearch
       environment:
         cluster.name: es-cluster
         node.name: es-node
         bootstrap.memory_lock: true
         discovery.type: single-node
-        ES_JAVA_OPTS: "-Xms512m -Xmx512m"
         xpack.security.enabled: true
+        xpack.license.self_generated.type: trial
         ELASTIC_PASSWORD: password
-    - image: docker.elastic.co/app-search/app-search:7.6.0
+    - image: docker.elastic.co/enterprise-search/enterprise-search:7.7.0
       name: appsearch
       environment:
         elasticsearch.host: http://elasticsearch:9200
         elasticsearch.password: password
-        APP_SEARCH_DEFAULT_PASSWORD: password
+        ENT_SEARCH_DEFAULT_PASSWORD: password
+        secret_management.encryption_keys: "[changeme]"
         allow_es_settings_modification: true
-        app_search.listen_host: 0.0.0.0
```

Comment:
Do we want to change this to `ent_search.listen_host`?

Reply:
I have removed it because it was not used anymore.